include/llama.h

```diff
@@ -304,8 +304,8 @@ extern "C" {
         uint32_t n_batch;           // logical maximum batch size that can be submitted to llama_decode
         uint32_t n_ubatch;          // physical maximum batch size
         uint32_t n_seq_max;         // max number of sequences (i.e. distinct states for recurrent models)
-        uint32_t n_threads;         // number of threads to use for generation
-        uint32_t n_threads_batch;   // number of threads to use for batch processing
+        int      n_threads;         // number of threads to use for generation
+        int      n_threads_batch;   // number of threads to use for batch processing
```

Comment:
Sorry for the confusion, but all types in the llama.cpp public API should be sized types to facilitate interop with other languages. Let's just revert this change and we will deal with the inconsistency between llama.cpp and ggml at other time.